client-src/elements/form-field-specs.ts

```diff
@@ -1461,7 +1461,12 @@ export const ALL_FIELDS: Record<string, Field> = {
     required: false,
     label: 'Origin trial contacts',
     help_text: html` List any other individuals or groups to include on the
-      contact list (e.g. for reminders on trial milestones).
+      contact list (e.g. for reminders on trial milestones). Mailing list emails
+      can be used here, but only email addresses of individuals will receive
+      access to view registrant data on
+      <a target="_blank" href="http://go/ot-registrants-dashboard"
```

Comment:
http://go/ot-registrants-dashboard is a Google specific URL (go/ links), not sure if this should be linked here

Reply:
We have a check in place to ensure only google and chromium accounts have access to the OT creation request form, so this already assumes that the user fulfills that request